pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -194,18 +197,17 @@ def set_distributed_mode(self, distributed_backend):
                     ' Trainer(distributed_backend=dp) (or ddp, ddp2).'
                     ' Setting distributed_backend=ddp_spawn for you.'
```

Comment:
change the warnings too with `BackendType.mode`?

Reply:
Trainer still works with strings...

Reply:
I meant using format strings and inject this new `BackendType` thing. It won't display `BackendType.mode` but its value. Just for consistency.